docs/conf.py

```diff
@@ -86,6 +87,8 @@
 nitpicky = True
 nitpick_ignore = []
 
+os.environ["FORCE_COLOR"] = "yes"  # Force --colored default value to be yes.
```

Comment:
can you make the comment lowercase and remove the period? generally, I don't write it as a sentence unless there's multiple sentences

Reply:
Sure (I just tend to follow pep8 by default).